src/main/java/uk/gov/hmcts/reform/hmc/domain/model/enums/DeleteHearingStatus.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.hmc.domain.model.enums;
+
+public enum DeleteHearingStatus {
+    HEARING_REQUESTED,
+    UPDATE_REQUESTED,
+    UPDATE_SUBMITTED,
+    AWAITING_LISTING,
+    LISTED;
+
+    public static boolean chekStatus(String status) {
```

Comment:
Typo in `chek`, but perhaps a more suitable name:
```suggestion
    public static boolean isValid(String status) {
```
Same comment applies to `UpdateHearingStatus`

Reply:
changed